### ISSUE TITLE
Lag spikes possibly related to the region.

### ISSUE THREAD

**Issue by MinallW** [feydan/satisfactory-tools#24]
Hello satisfactory community!,

Thanks for this amazing project, I've able to shoot my own server, using eu-west-3 as I'm located in Spain - Madrid.

Now, I wonder if there's something to configure or, is there some known stability network issues regarding eu-west-3 Paris (can't find much about it), since without changing any configuration, I get a normal 50ms, but when doing something it will become rapidly into 150>, then 300> and even with a friend, into 2000ms. This is not normal behaviour of course, and makes it unplayable.

I really wonder if this is more about the CPU, which I see on 'top' to go around 60 to 100 to 110 % and so on. There's RAM available. But I guess the CPU fluctuating is normal of course.

So I wonder if this power spikes are because of network stability, and I should change my region to a more stable one (I wonder which one), or if I should upgrade de CPU (instance-type).

Thanks!

**Comment by ThomasVictoria** [comment]
What instance are you using ? Since this project was started, resources requirement increased a bit, and it might be the problem. I'm playing in eu-west-3 as well but don't notice anything with my current configuration.

Also see this [PR](https://github.com/feydan/satisfactory-server-aws/pull/23), I made some tests and made a small report.

**Comment by MinallW** [comment]
Yes, it was because of the instance. After upgrading to a better CPU (since having RAM available but still not being able to compute was my issue), the problem went away and the server went flawlessly

**Comment by christian-franklin** [comment]
What CPU and RAM selection did you pick? I'm noticing lag issues on v1.0 with 2 people on an m6a.xlarge which is 4 CPUs and 16GB of ram.

Edit: Also tried r7i.xlarge (4 CPUs and 32 GB of ram) and still experience lag. Tried 2 different AWS regions as well.
